### ISSUE TITLE
Not working in newest Beta Version of Home assistant

### ISSUE THREAD

**Issue by ShadyMCShadesen** [UI-Lovelace-Minimalist/UI#1553]
**Describe the bug**
UI-Lovelace-Minimalist not working in newest beta version of home assistant (2024.12.X)

**To Reproduce**
Update to newest beta version and restart home assistant

**Expected behavior**
The overlay to work

**Related Logs** (can be found in the browser-console (F12))
Logger: homeassistant.setup
Quelle: setup.py:269
Erstmals aufgetreten: 07:27:19 (1 Vorkommnisse)
Zuletzt protokolliert: 07:27:19

Setup failed for custom integration 'ui_lovelace_minimalist': Requirements for ui_lovelace_minimalist not found: ['aiofiles==0.8.0'].

**Comment by wokkeltje13** [comment]
Fixed with https://github.com/UI-Lovelace-Minimalist/UI/pull/1543

**Comment by fear** [comment]
im having the same issue. Since i have no longer a working ui, is there a way to Cherrypick tihs request? ^^

I accidantaly installed the beta on production... *blame*

**Comment by ShadyMCShadesen** [comment]
A hotfix would be awesome

**Comment by DavidGeorge528** [comment]
Any movement on this? The stable release of 2024.12.0 will be out any day and Minimalist UI is still incompatible

**Comment by ShadyMCShadesen** [comment]
> Any movement on this? The stable release of 2024.12.0 will be out any day and Minimalist UI is still incompatible

Yeah idk what's wrong with the developers, there are no hotfixes, I can only imagine how many are currently without a dashboard 🥲

**Comment by Calimerorulez** [comment]
I just edited 

custom_components/ui_lovelace_minimalist/manifest.json

and removed the `aiofiles==0.8.0 line`, rebooted Home Assistant, and it works again.

**Comment by JFS-HA** [comment]
> I just edited
> 
> `custom_components/ui_lovelace_minimalist/manifest.json`
> 
> and removed the `aiofiles==0.8.0 line`, rebooted Home Assistant, and it works again.

THX!!!

**Comment by majorhector** [comment]
> I just edited
> 
> `custom_components/ui_lovelace_minimalist/manifest.json`
> 
> and removed the `aiofiles==0.8.0 line`, rebooted Home Assistant, and it works again.

Thank you a lot. You are my savior 👍

**Comment by warmfire540** [comment]
I pinged @silamon who made the PR on #1543 - he may need to tweak the PR, or not sure if the repository owner cares.  if they don't answer in a few days i can make a diff PR into release branch 🤷‍♀️

**Comment by KruseLuds** [comment]
> I just edited
> 
> `custom_components/ui_lovelace_minimalist/manifest.json`
> 
> and removed the `aiofiles==0.8.0 line`, rebooted Home Assistant, and it works again.
This resolves my issue as well.

**Comment by KruseLuds** [comment]
> I just edited
> 
> `custom_components/ui_lovelace_minimalist/manifest.json`
> 
> and removed the `aiofiles==0.8.0 line`, rebooted Home Assistant, and it works again.

This resolves the issue for me as well!

**Comment by KruseLuds** [comment]
> I pinged @silamon who made the PR on #1543 - he may need to tweak the PR, or not sure if the repository owner cares. if they don't answer in a few days i can make a diff PR into release branch 🤷‍♀️

Please do!

**Comment by MastrUsr** [comment]
No need to remove it, just change it to from == to >=
.../homeassistant/custom_components/ui_lovelace_minimalist/manifest.json

From:
    "requirements": [
        "aiofiles>=0.8.0",

To:
    "requirements": [
        "aiofiles==0.8.0",


Full file:

```
{
    "codeowners": [
        "@stokkie90"
    ],
    "config_flow": true,
    "dependencies": [
        "lovelace",
        "http",
        "frontend"
    ],
    "documentation": "https://ui-lovelace-minimalist.github.io/UI/",
    "domain": "ui_lovelace_minimalist",
    "iot_class": "calculated",
    "issue_tracker": "https://github.com/UI-Lovelace-Minimalist/UI/issues",
    "name": "UI Lovelace Minimalist",
    "requirements": [
        "aiofiles>=0.8.0",
        "aiogithubapi>=22.2.4"
    ],
    "version": "v1.3.10"
}

```

**Comment by charles-moody** [comment]
I don’t want to be a nitpicker, but you got the example wrong, though the code itself is right.

From:
aiofiles==0.8.0

To:
aiofiles>=0.8.0

Anyway, it’s clear from the thread.

Thanks a lot, it’s working again now!

**Comment by MastrUsr** [comment]
> I don’t want to be a nitpicker, but you got the example wrong, though the code itself is right.
> 
> From: aiofiles==0.8.0
> 
> To: aiofiles>=0.8.0
> 
> Anyway, it’s clear from the thread.
> 
> Thanks a lot, it’s working again now!

Thanks, changed it 😅

**Comment by markmghali** [comment]
I am on the official release of 2024.12 and it is also not working

**Comment by ShadyMCShadesen** [comment]
> I am on the official release of 2024.12 and it is also not working  

How should it, last update of Ui Lovelace Minimalist was months ago

**Comment by ShadyMCShadesen** [comment]
Man I don't care about the developer.
My point was that it obviously still doesn't work in 2024.12, if it didn't work in the beta and no update has been released since then.

**Comment by ShadyMCShadesen** [comment]
Man I don't care about the developer.
My point was that it obviously still doesn't work in 2024.12, if it didn't work in the beta and no update has been released since then.

**Comment by warmfire540** [comment]
Juste delete it, keep it simple.  Restart HA after to pick up the change.

should be this after, just like the PR fix
```json
  "requirements": [
    "aiogithubapi>=22.2.4"
  ],
```

please don't comment if you're not having issues just to chime in as it notifies everyone.

it will look like this after, use VSCode or file editor add ons to edit

<img width="1281" alt="image" src="https://github.com/user-attachments/assets/ec9016f7-8899-4f9f-aee2-4f0d0b310c75">

**Comment by xavierlaffargue** [comment]
Excuse me but why is so long to release a new version with this hotfix (with only one character to change!), I think that the process is not optimal...

**Comment by KruseLuds** [comment]
Because evidently there is now not a maintainer for this codebase

**Comment by jamesmyatt** [comment]
Please don't be rude. The maintainers are ordinary people doing this in their spare time for the good of the community. There are a million very good reasons why they haven't released this yet. Being rude is not going to get it done faster; might even take longer. If you don't like it, then ask for your money back and go do something else. No-one is forcing you to keep using it.

The manual patch (https://github.com/UI-Lovelace-Minimalist/UI/issues/1553#issuecomment-2519972022) will literally take less time than reading this comment.

**Comment by wilbiev** [comment]
Issue solved in new available release v1.3.11
